dev/start

```diff
@@ -1,45 +1,164 @@
 #!/usr/bin/env sh
 # NOTE: This script should be sourced! The shebang is only here to help syntax highlighters.
 
-DEVENV="${1:-./devenv}"
-PYTHON="${PYTHON:-3.8}"
-_CONDA="${DEVENV}/bin/conda"
-
-if ! [ -f "$DEVENV/conda-meta/history" ]; then
-    if [ "$(uname)" = Darwin ]; then
-        if [ ! -f miniconda3.sh ]; then
-            curl "https://repo.anaconda.com/miniconda/Miniconda3-latest-MacOSX-x86_64.sh" -o miniconda3.sh
-        fi
-        bash miniconda3.sh -bfp "$DEVENV"
-        rm -f miniconda3.sh
-        "${_CONDA}" install -yq -p "$DEVENV" python="$PYTHON" --file tests/requirements.txt -c defaults
-    elif [ "$(uname)" = Linux ]; then
-        if [ ! -f miniconda3.sh ]; then
-            curl "https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh" -o miniconda3.sh
+if ! (return 0 2> /dev/null); then
+    echo "ERROR: Source this script: . '$0'." >&2
```

Comment:
```suggestion
    echo "ERROR: Source this script: source '$0'." >&2
```
Let's use source here since that'll be harder to miss than the canonical dot :)